OutOfSchool/OutOfSchool.DataAccess/Models/DateTimeRange.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+
+namespace OutOfSchool.Services.Models
+{
+    public class DateTimeRange
+    {
+        public long Id { get; set; }
+
+        public TimeSpan StartTime { get; set; }
+
+        public TimeSpan EndTime { get; set; }
+
+        public long WorkshopId { get; set; }
+
+        public virtual List<Workday> Workdays { get; set; }
```

Comment:
Suggestion:
Variant **0**. Leave as is
Variant **1**. instead of `public virtual List<Workday> Workdays { get; set; }`
use
```
public bool IsMonday {get; set;}
...
public bool IsSunday {get; set;}
```
and perform mapping to `List<DayOfWeek>` in `automapper` profile config

Variant **2**. instead of `public virtual List<Workday> Workdays { get; set; }`
use `public byte WorkDaysMask {get; set;}`
and perform mapping to `List<DayOfWeek>` in `automapper` profile config

Variant **3**. instead of `public virtual List<Workday> Workdays { get; set; }`
use `public byte WorkDaysMask {get; set;}`
and perform mapping to custom `DaysOfWeek` flag `Enum` in `automapper` profile config